src/top-bar/menu/menu.js

```diff
@@ -52,8 +57,9 @@ Menu.propTypes = {
 export default function Menu({ tabs, activeTab, setActiveTab }) {
   const classes = useStyles(useMuiStyles);
   const { isExtraSmallScreen, isSmallScreen, isLargeScreen } = useScreenSize();
-
-  const showNewsletterButton = !isExtraSmallScreen && !isSmallScreen;
+  let newsletterSubcription = getNewsletterSubcription();
```

Comment:
- Let's use constants where possible.
- `isSubscribedToNewsletter` is a more descriptive name.